catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -266,6 +273,15 @@ private SchematronReport generateReport(String metadata, Templates validator)
         return report;
     }
 
+    /**
+     * Replace tabs, literal carriage returns, and newlines with a single whitespace
+     * @param input
+     * @return
+     */
+    static String sanitize(final String input) {
```

Comment:
Sharing code between the implementation and the test code usually defeats the purpose of the test. What if someone changes the regex by mistake and breaks the way the sanitize works? How would you know it's broken?